executor/predictor/predictor_process_test.go

```diff
@@ -455,3 +461,16 @@ func TestModelWithLogResponses(t *testing.T) {
 	g.Expect(smRes.GetData().GetNdarray().Values[1].GetNumberValue()).Should(Equal(2.0))
 	g.Eventually(func() bool { return logged }).Should(Equal(true))
 }
+
+func TestPredictNilPUIDError(t *testing.T) {
+	t.Logf("Started")
+	g := NewGomegaWithT(t)
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	})
+	server := httptest.NewServer(handler)
+	defer server.Close()
+	graph := &v1.PredictiveUnit{}
+	_, err := createPredictorProcessWithoutPUIDInContext(t).Predict(graph, createPredictPayload(g))
+	g.Expect(err).NotTo(BeNil())
+	g.Expect(err.Error()).Should(Equal("context value Seldon PUID Header is nil: interface to string conversion failed"))
```

Comment:
The message should be turned into a `const` if we are going to use it in multiple places.

Reply:
Fixed. See #1841 for follow up changes